lib/ClangImporter/ClangImporter.cpp

```diff
@@ -751,10 +751,21 @@ void importer::getNormalInvocationArguments(
   invocationArgStrs.push_back("-fapinotes-modules");
   invocationArgStrs.push_back("-fapinotes-swift-version=" +
                               languageVersion.asAPINotesVersionString());
-  invocationArgStrs.push_back("-iapinotes-modules");
-  invocationArgStrs.push_back((llvm::Twine(searchPathOpts.RuntimeResourcePath) +
-                               llvm::sys::path::get_separator() +
-                               "apinotes").str());
+  std::string apiNotesPath = (llvm::Twine(searchPathOpts.RuntimeResourcePath) +
+                              llvm::sys::path::get_separator() +
+                              "apinotes").str();
+  auto vfs = ctx.SourceMgr.getFileSystem();
+  if (!vfs->exists(apiNotesPath)) {
```

Comment:
If the path exists, I think that is okay, and we should just be doing a second additional path.